src/controller/CompanyProfileController.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, Response } from "express"
+import { CompanyProfile } from "../entity/CompanyProfile"
```

Comment:
can we separate imports from 3rd parties and from our project?

import express
import from project
->
import express 

import from project
?